### ISSUE TITLE
Image doesn’t fit in the viewport after rotating

### ISSUE THREAD

**Issue by andrew--r** [sachinchoolur/lightGallery#1171]
When rotating image, it may not fit in the viewport so it becomes cropped. Rotating should also scale image to fit it in the viewport.

<details>
<summary>Normal</summary>

![Screenshot 2021-04-06 at 12 20 57](https://user-images.githubusercontent.com/2734841/113689134-a7f3bf80-96d2-11eb-9d86-bc16b05a5456.png)

</details>

<details>
<summary>Rotated</summary>

![Screenshot 2021-04-06 at 12 21 00](https://user-images.githubusercontent.com/2734841/113689130-a5916580-96d2-11eb-8f18-1778d115e64d.png)

</details>

**Comment by sachinchoolur** [comment]
Yes, Currently this is a limitation with the rotate plugin. I've already started working on improving zoom/rotate modules. I'll try to include the fix in version 2.4 or 2.5

**Comment by sachinchoolur** [comment]
BTW. Are you still able to access lightGallery 1.x website?

**Comment by andrew--r** [comment]
> I've already started working on improving zoom/rotate modules. I'll try to include the fix in version 2.4 or 2.5

Thanks!

> BTW. Are you still able to access lightGallery 1.x website?

Nope, I just copied issue from lg-rotate plugin repo since the previous issue hasn’t been answered:

https://github.com/sachinchoolur/lg-rotate/issues/1
